fre/make/tests/null_example/compile.yaml

```diff
@@ -0,0 +1,48 @@
+compile:
+     experiment: "null_model_full"
+     container_addlibs: 
+     baremetal_linkerflags:  
+     src:
+          - component: "FMS"
```

Comment:
It's just cause FMS and the coupler aren't specific to the null_model, everything else has the null cause they don't do anything.

Reply:
What about `ice_param`? Should it have `_null`? Also, since FMS and the coupler aren't null, will the model compile these?

Reply:
I think theres no null alternative for ice_param so the real version is required.

Yeah the whole point of the null model is to compile the coupler program without real components, since its the main program thats pulling all these other modules in.  It still needs FMS cause it tests basic things every model does like reading certain input files and mpi stuff.

Reply:
Ah ok, thanks!